src/Monolog/Formatter/HtmlFormatter.php

```diff
@@ -50,6 +50,7 @@ public function __construct(?string $dateFormat = null)
      * @param string $th       Row header content
      * @param string $td       Row standard cell content
      * @param bool   $escapeTd false if td content must not be html escaped
+     * @return string
```

Comment:
This is useless, there is already a PHP type hint on the method.
This should be reverted,

Reply:
Yes.. I added there for consistency with other docblocks.. I saw that where ever a docblock existed it had the return typehint.. I will revert...

Reply:
It's only added when it provides a description or sometimes because it was there in 1.x and was missed when converting. e.g. addTitle below has a return that should be moved to the method signature.

Reply:
> addTitle below has a return that should be moved to the method signature.

I did that too.

>  I will revert...

reverted.